src/verilog/parser.y

```diff
@@ -873,16 +903,21 @@ module_or_generate_item_declaration:
 	;
 
 non_port_module_item:
-	  attribute_instance_brace generate_region { $$=$2; }
+	  attribute_instance_brace generate_region
+		{ add_attributes($2, $1); $$=$2; }
         | module_or_generate_item
-        | attribute_instance_brace specparam_declaration {$$=$2; }
-	| attribute_instance_brace specify_block { $$=$2;}
+        | attribute_instance_brace specparam_declaration
+		{ add_attributes($2, $1); $$=$2; }
+	| attribute_instance_brace specify_block
+		{ add_attributes($2, $1); $$=$2; }
         ;
 
 /*
 	  module_or_generate_item
-	| attribute_instance_brace parameter_declaration { $$=$2; }
-	// | attribute_instance_brace local_parameter_declaration { $$=$2; }
+	| attribute_instance_brace parameter_declaration
+	//	{ add_attributes($2, $1); $$=$2; }
+	// | attribute_instance_brace local_parameter_declaration
+	//	{ add_attributes($2, $1); $$=$2; }
 	;
 */
```

Comment:
Why was/is this commented out?

Reply:
These are not in the 1800-2017 grammar -- will clean out.